Oanda/modules/trader/interpreter/interpreter.py

```diff
@@ -220,9 +228,20 @@ def trade(self, prediction, latest_value):
         filter_dict(data)
         
         print(readable_output(data))
+        
         try:
-            OrdersOrderCreate(self.access_token, self.accountID, data=data)
-            print("Bought ", units, " ", self.instrument, " value of trade: ", units*latest_value)
+            for i in range(tries):
+                if i < tries:
```

Comment:
If `tries=3`, `i` will loop through the list `[0, 1, 2]`. As a result, `i<tries` will always be `True`.